### ISSUE TITLE
Should not lint on R6 object instantiation

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#22]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  R6[R6Class],
)

SomeClass <- R6Class("SomeClass",   # nolint
  public = list()
)

s <- SomeClass$new()
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result

```
<text>:10:6: warning: [box_usage_linter] <package/module>$function does not exist.
s <- SomeClass$new()
     ^~~~~~~~~~~~~
```

## Expected result

Should not lint on R6 object instantiation.
